docs/UserGuide.adoc

```diff
@@ -177,15 +176,81 @@ There is no need to save manually.
 
 == Command Summary
 
-* *Add* `add n/NAME p/PHONE_NUMBER e/EMAIL a/ADDRESS [t/TAG]...` +
-e.g. `add n/James Ho p/22224444 e/jamesho@example.com a/123, Clementi Rd, 1234665 t/friend t/colleague`
-* *Clear* : `clear`
-* *Delete* : `delete INDEX` +
-e.g. `delete 3`
-* *Find* : `find KEYWORD [MORE_KEYWORDS]` +
-e.g. `find James Jake`
-* *List* : `list`
-* *Help* : `help`
-* *Select* : `select INDEX` +
-e.g.`select 2`
-* *History* : `history`
+=== View help
+
+Format: `help`
+
+=== Adding a task
+
+Format: `add n/[task name] s/[time] d/[date] t/[tags]`
+
+Examples:
+****
+* `add n/complete tutorial s/2200 d/22/03/16 t/school`
+* `add n/complete tutorial s/2200 d/22/03/16`
+* `add n/complete tutorial`
+* `add n/complete tutorial s/2100`
+****
+
+=== Editing a Task
+
+Format: `edit [index] [prefix][new value]`
+
+Examples:
+****
+* `edit 1 n/tutorial` +
+Edits the task name to tutorial.
+* `edit 2 s/2200` +
+Edits the task's time to 2200.
+* `edit 2 s/2200 d/21/03/17` +
+Edits the task's time to 2200 and the date to 21/03/17.
+****
+
+=== Finding a Task
+
+Format: `find [keyword(s)]`
+
+Examples:
+****
+* `find tutorial` 
```

Comment:
Maybe you can add all possible types of search optmisations you have? eg. partial word search or case insensitive search.